app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -22,7 +32,7 @@ class Identification extends React.Component {
       current: { ...this.props.currentState },
       errors: {},
       modified: {},
-      languageOptions: this.getLanguageOptions(),
+      primaryLanguageMessage: this.getPrimaryLanguageMessage(tryToMatchLanguage(this.props.currentState.patient.primary_language)),
```

Comment:
I'd still prefer to not keep this in state but will defer to others

Reply:
done.